protocol/avdl/stellar1/remote.avdl

```diff
@@ -25,18 +26,33 @@
     keybase1.KID KID;  // public key data was encrypted for
   }
 
+  record Members {
+    AccountID fromStellar;
+    string fromKeybase;
+    keybase1.UID fromUID;
+    keybase1.DeviceID fromDeviceID;
+    AccountID toStellar;
+    string toKeybase;
+    keybase1.UID toUID;
+  }
+
   record TransactionSummary {
     TransactionID stellarID;
     KeybaseTransactionID keybaseID;
+    string status;
+    string errMsg;
 
     EncryptedNote note;
 
     Asset asset;
     string amount;
-    AccountID stellarFrom;
-    AccountID stellarTo;
-    keybase1.UID keybaseFrom;
-    keybase1.UID keybaseTo;
+    string displayAmount;
+    string displayCurrency;
```

Comment:
what's this?

Reply:
Once we look up exchange rate, this will be what the user sees in GUI "$5 USD" for sending 27.234XLM.

Reply:
I bet they'd want more structured data. But we can just change it on demand if these are short lived objects.

Reply:
Yeah, mainly placeholders at this point.